### PR TITLE
COPY EP Letters of support

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 1]
```diff
@@ -9,32 +9,30 @@ def promotion_step4
...
 
-            <br>
+          )
 
+          pdf_context %(
             <p>
-              <strong>
-                Who should I choose to write letters of support?
-              </strong>
+              As nominator you have sole responsibility for sourcing at least 2 letters of support. Nominations with less than 2 letters will not be accepted. We recommend asking 3-5 supporters to ensure you meet the minimum requirements, as some people might not reply.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [270/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 2]
```diff
@@ -9,32 +9,30 @@ def promotion_step4
...
+            </p>
+            <p>
+              If you have supporters who would rather write a hard copy letter, please see question D2.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [103/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 3]
```diff
@@ -9,32 +9,30 @@ def promotion_step4
...
             </p>
+          )
 
+          hint "Who should I choose to write letters of support?", %(
             <p>
               Letters of support should be from those with first-hand knowledge of the nominee’s contribution to enterprise promotion and the impact their work has had on others. This could be colleagues, collaborators, or those who have benefitted from their work. You, the nominator, may not write a letter of support.
             </p>
             <p>
-              At least one letter should be from a large organisation (e.g. employer, nonprofit, college, local authority) able to provide an assessment of the extent to which the nominee’s contribution sets them aside from their peers.
+              At least one letter should be from a large organisation (eg. employer, nonprofit, college, local authority) able to provide an assessment of the extent to which the nominee’s contribution sets them aside from their peers.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [235/100]

**Comment by RuslanHamidullin** [comment]
Best I saw in live :100:

**Pull request merged**